Samples/BackgroundBlur/BackgroundBlur/common/GrowArray.h

```diff
@@ -0,0 +1,122 @@
+//-----------------------------------------------------------------------------
+// File: GrowArray.h
+// Desc: Growable array class.
+//
+// THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF
+// ANY KIND, EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO
+// THE IMPLIED WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A
+// PARTICULAR PURPOSE.
+//
+//  Copyright (C) Microsoft Corporation. All rights reserved.
+//-----------------------------------------------------------------------------
+
+
+#pragma once
+
+
+namespace MediaFoundationSamples
+{
+
+    // Class template: Re-sizable array. 
```

Comment:
If there's a reason not to use std::vector, add it to the comments here

Reply:
There isn't- in fact the demo doesn't use growarray at all. I'll remove it.